docs/www/acls.md

```diff
@@ -56,61 +62,160 @@ redpanda:
 
-> For an in-depth guide about enabling TLS and mutual TLS in redpanda check out [our guide on the subject](https://vectorized.io/blog/tls-config/).
+For an in-depth guide about enabling TLS and mutual TLS in redpanda check out [our guide on the subject](https://vectorized.io/blog/tls-config/).
 
 ### SASL/ SCRAM
 
 Redpanda also supports client authentication via SASL/SCRAM (that is, the Simple Authentication and Security Layer protocol, using the Salted Challenge Response Authentication Mechanism), which is based on usernames & passwords.
 
-To enable it, set `redpanda.enable_sasl` to `true` in the configuration file, as well as list at least one "superuser" which after created will have permissions for all operations on the clusters.
+To enable it, set `redpanda.enable_sasl` to `true` in the configuration file, as well as list at least one "superuser" which after being created will have permissions for all operations on the clusters.
+
+You config must look something like this:
 
 ```yaml
 redpanda
   
   enable_sasl: true
   superusers:
-  - admin
+    - username: admin
   # The rest of the config...
 ```
 
+<tabs>
+  <tab id="Local Redpanda">
+  You can find the configuration file here
+
+  ```bash
+  /etc/redpanda/redpanda.yaml
+  ```
+
+After you change it's value, restart Redpanda service for changes to be in effect. 
+
+  </tab>
+  <tab id="Docker Container">
+  To access the files inside the container, first you have to enter the container.
+
+  You can do that by running:
+
+```bash
+  docker exec -it <name-of-container> bash
+```
+
+Change <name-of-container> for your container and execute the command. 
+Then go to the same directory as a local redpanda config:
+
+```bash
+  /etc/redpanda/redpanda.yaml
+```
+
+After you finish it, restart the container for changes to be in effect. 
+
+  </tab>
+  <tab id="Kubernetes">
+  For Kubernetes, things are a little different. You can change your configuration file with the YAML for the cluster.
+
+  If you got our [one-cluster sample](https://github.com/vectorizedio/redpanda/blob/dev/src/go/k8s/config/samples/one_node_cluster.yaml) you should add these values to enable different flags.
+
+  For example: 
+
+```yaml
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: external-connectivity
+spec:
+  image: "vectorized/redpanda"
+  version: "latest"
+  replicas: 3
+  enableSasl: true
+  superUsers:
+    - username: admin
+  resources:
+    requests:
+      cpu: 1
+      memory: 2Gi
+    limits:
+      cpu: 1
+      memory: 2Gi
+  configuration:
+    rpcServer:
+      port: 33145
+    kafkaApi:
+     - port: 9092
+     - external:
+         enabled: true
+    pandaproxyApi:
+     - port: 8082
+     - external:
+         enabled: true
+    adminApi:
+    - port: 9644
+    developerMode: true
+```
+
+**_Note_** - The attributes in K8s YAML are in camelCase instead of snake_case used in local redpanda.yaml
```

Comment:
`>` indents the note

```suggestion
> **_Note_** - The attributes in Kubernetes YAML are in camelCase instead of snake_case used in the local redpanda.yaml
```